main.rb

```diff
@@ -0,0 +1,30 @@
+require './app'
+
+class Main < App
+  def run
+    puts 'Your most welcome to OOP School Library'
+    loop do
+      list
+      option = gets.chomp
+
+      break if option == '6'
+      # option.gsub(/[^0-9]/, '')
+      # get_num option
+    end
+    puts 'Thanks for visiting our library'
+  end
+
+  def list
+    puts
+    puts 'Kindly select an option by entering a number'
+    puts '0 - List all books'
+    puts '1 - List all people'
+    puts '2 - Create a person'
+    puts '3 - Create a book'
+    puts '4 - Create a rental'
+    puts '5 - List all rentals for a given person id'
+    puts '6 - Exit'
+  end
```

Comment:
- [ ] I noticed that when I select a menu for example create a person it returns a menu instead of the menu to create a person which means I can't create a person,book, also a rental. Please kindly fix this issue, you can see how the application should work in this [video.](https://www.youtube.com/watch?v=vkkgrhD6aXQ) 🙃 🙃

<img width="990" alt="Screen Shot 2022-09-08 at 17 07 57" src="https://user-images.githubusercontent.com/74501523/189160943-cfbd1b8d-8a9d-4252-ad4f-a4a27cc6060c.png">